torch/nn/modules/module.py

```diff
@@ -1574,8 +1580,16 @@ def _call_impl(self, *args, **kwargs):
 
     __call__ : Callable[..., Any] = _call_impl
 
+    def __getstate__(self):
+        state = self.__dict__.copy()
+        state.pop("_compiled_call_impl", None)
+        return state
+
     def __setstate__(self, state):
         self.__dict__.update(state)
+        if "_compiled_call_impl" in self.__dict__:
```

Comment:
we can potentially also compile a model when we load it but then we'd also need to save all args and kwargs that were passed in to torch.compile - I'm debating whether this is worth it

Reply:
I'm not sure what this is doing? Why would you want to reset the currently compiled model when doing setstate?
- How did you manage to compile your model before this? This happens on torch.load
- Is that invalidating the currently compiled call, why would we do that?

Reply:
Just removed it